src/gluonts/model/n_beats/_network.py

```diff
@@ -666,7 +707,12 @@ def __init__(self, *args, **kwargs) -> None:
 
     # noinspection PyMethodOverriding,PyPep8Naming
     def hybrid_forward(
-        self, F, past_target: Tensor, future_target: Tensor = None
+        self,
+        F,
+        past_target: Tensor,
+        past_observed_values: Tensor,
+        future_target: Optional[Tensor] = None,
+        future_observed_values: Optional[Tensor] = None,
```

Comment:
I'm wondering if this block could just avoid having these inputs, since they aren't used. I know that `future_target` was there already, but maybe that could be removed too

Reply:
it seems that the signature of hybrid_forward must match the one of the super class (some tests fail if this is not the case). That's why probably all four inputs are needed for the hybrid_forward of all the 3 classes.

Reply:
Ah I see, it's probably `mypy` who's complaining?

Reply:
Can't remember, might be.